modules/persistent-storage-local-discovery.adoc

```diff
@@ -0,0 +1,167 @@
+// Module included in the following assemblies:
+//
+// storage/persistent_storage/persistent-storage-local.adoc
+
+[id="local-storage-discovery_{context}"]
+= Automating discovery and provisioning for local storage devices
+
+The Local Storage Operator automates local storage discovery and provisioning. With this feature, you can simplify installation when dynamic provisioning is not otherwise available during deployment, such as with bare metal, VMware, or AWS store instances with attached devices.
+
+[IMPORTANT]
+====
+Automatic discovery and provisioning is a Technology Preview feature only. Technology Preview features are not supported with Red Hat production service level agreements (SLAs) and might not be functionally complete. Red Hat does not recommend using them in production. These features provide early access to upcoming product features, enabling customers to test functionality and provide feedback during the development process.
+
+For more information about the support scope of Red Hat Technology Preview features, see https://access.redhat.com/support/offerings/techpreview/.
+====
+
+Use the following procedure to create discovery requests, automatically discover local devices, and automatically provision local volumes for selected devices.
+
+.Prerequisites
+
+* The Local Storage Operator is installed.
+
+* Local disks are attached to {product-title} nodes.
+
+* You have access to the {product-title} web console and the `oc` command-line interface (CLI).
+
+.Procedure
+. To create a local volume discovery request from the web console:
+
+.. Navigate to *Operators* -> *OperatorHub* -> *Installed Operators*.
```

Comment:
Does this procedure need to be done from the Administrator perspective? If so, you could update this first step as follows:

 .. In the *Administrator* perspective, navigate to *Operators* -> *OperatorHub* -> *Installed Operators*.

Reply:
The same goes for the other items in the procedure where appropriate.

Reply:
@chao007 or @gnufied do you know?

Reply:
Yes creation of any LSO related resources requires admin access.

Reply:
Added Paul's suggestion to the first step of the procedure, which is a different one now that some things got moved around.